src/libs/ReportSecondaryActionUtils.ts

```diff
@@ -43,16 +42,7 @@ import {
     isWorkspaceEligibleForReportChange,
 } from './ReportUtils';
 import {getSession} from './SessionUtils';
-import {
-    allHavePendingRTERViolation,
-    getOriginalTransactionWithSplitInfo,
-    hasReceipt as hasReceiptTransactionUtils,
-    isCardTransaction as isCardTransactionUtils,
-    isDuplicate,
-    isOnHold as isOnHoldTransactionUtils,
-    isPending,
-    isReceiptBeingScanned,
-    shouldShowBrokenConnectionViolationForMultipleTransactions,
+import {allHavePendingRTERViolation, isCardTransaction as isCardTransactionUtils, isDuplicate, isOnHold as isOnHoldTransactionUtils, shouldShowBrokenConnectionViolationForMultipleTransactions
```

Comment:
FYI added `isCardTransaction as isCardTransactionUtils,` for https://github.com/Expensify/App/pull/62997 which was just recently merged to `main`

Reply:
oh this did break prettier 🤷

Reply:
merp yarp... I can fix that on `main` after the revert? 🙏  i'd love to just get this merged so we don't have to deal with any more conflicts haha